mempool/src/shared_mempool/tasks.rs

```diff
@@ -104,18 +106,23 @@ pub(crate) async fn process_transaction_broadcast<V>(
     request_id: Vec<u8>,
     timeline_state: TimelineState,
     peer: PeerNetworkId,
+    timer: HistogramTimer,
 ) where
     V: TransactionValidation,
 {
+    timer.stop_and_record();
```

Comment:
We we actually want to stop the time right when entering the function? I thought these timers were for execution time or is that captured by the timer below?

Reply:
yes, so the timer passed in would measure the time b/w this task being scheduled by the bounded executor and actually started. the timer below is how the latency of this started task itself